_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -76,7 +76,12 @@ struct Handlers {
         QueryIOSurfFunc queryIOSurf;
     };
 
+    struct Fallback {
+        CtorFunc        ctor;
```

Comment:
Unfortunately, this works only since C++14, as for C++11 [aggregate initialization can't be used if a class has member-initializers](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3605.html). I'll try to use std::function instead.